scripts/replicate_tests_db.coffee

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env coffee
+__ = require('config').universalPath
+_ = __.require 'builders', 'utils'
+breq = require 'bluereq'
+{ exec } = require 'child_process'
+{ Promise } = __.require 'lib', 'promises'
+error_ = __.require 'lib', 'error/error'
+
+dbHost = require('config').db.fullHost()
+allDbsUrl = dbHost + '/_all_dbs'
+
+tests_dbs = (db)-> db.includes '-tests'
+dbUrl = (dbName)-> "#{dbHost}/#{dbName}"
+dbsBaseNames = Object.keys __.require('couch', 'list')
+testDbsNames = dbsBaseNames.map (dbBaseName)-> "#{dbBaseName}-tests"
+
+Promise.all dbsBaseNames
+.map (dbName)->
+  dbTestName = "#{dbName}-tests"
+  repDoc =
+    source: dbUrl dbTestName
+    target: dbUrl dbName
+  breq.post "#{dbHost}/_replicate", repDoc
+  .then (res)-> res.body
+  .then _.Log "#{dbTestName} replication response"
+.catch (err)->
+  console.log "#{err.body.reason}\n\
+  Hum, have you ran the tests first ?"
```

Comment:
shouldn't `scripts/populate_fixtures` make sure tests are run instead?

Reply:
this script replicate dbs, its the user responsability to replicate dbs that exist already. It is written in the documentation where [this](https://github.com/inventaire/docker-inventaire/blob/master/README.md#fixtures) script is described